<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>547</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>System monitor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/img/icon.png</normaloff>:/img/icon.png</iconset>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>731</width>
      <height>501</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tabProc">
     <attribute name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/img/process.png</normaloff>:/img/process.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>430</y>
        <width>111</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Kill process</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/img/kill.png</normaloff>:/img/kill.png</iconset>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>701</width>
        <height>411</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>430</y>
        <width>121</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/img/refresh.png</normaloff>:/img/refresh.png</iconset>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabCPU">
     <attribute name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/img/processor.png</normaloff>:/img/processor.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>701</width>
        <height>441</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabOS">
     <attribute name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/img/os.png</normaloff>:/img/os.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QLabel" name="labelOS">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>631</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Operating system:    </string>
      </property>
     </widget>
     <widget class="QLabel" name="labelKernel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>661</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Kernel:     </string>
      </property>
     </widget>
     <widget class="QLabel" name="labelImgOS">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>130</y>
        <width>551</width>
        <height>311</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabMem">
     <attribute name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/img/memory.png</normaloff>:/img/memory.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QProgressBar" name="progressBarMem">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>341</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBarSwp">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>90</y>
        <width>341</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Memory usage:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Swap usage:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelMemOf">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>30</y>
        <width>201</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelSwpOf">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>90</y>
        <width>211</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>753</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
